fairroot/tools/FairBaseTypes.h

```diff
@@ -0,0 +1,28 @@
+/********************************************************************************
+ * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairBaseTypes.h
+ *
+ *  Created on: May 16, 2023
+ *      Author: R.Karabowicz
+ */
+
+#ifndef BASE_FAIRTYPES_H_
+#define BASE_FAIRTYPES_H_
+
+#include <Rtypes.h>
+
+namespace fr {
```

Comment:
```suggestion
namespace fairroot {
```

What about a more descriptive namespace?
Yes, this feels like more typing. But really, it's 2023 and we have good editors.